<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-QEV1T8U" ignored="0" name="BankingApp Test(failed)(failed)" tests="2" failures="2" timestamp="24 Jul 2021 12:11:35 GMT" time="32.576" errors="0">
  <testcase name="loginDDT" time="7.438" classname="com.inetbanking.testCases.Tc_LoginDDT_002">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at com.inetbanking.testCases.Tc_LoginDDT_002.loginDDT(Tc_LoginDDT_002.java:33)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="6.525" classname="com.inetbanking.testCases.Tc_LoginDDT_002">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at com.inetbanking.testCases.Tc_LoginDDT_002.loginDDT(Tc_LoginDDT_002.java:33)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
</testsuite> <!-- BankingApp Test(failed)(failed) -->
